service_darwin.go

```diff
@@ -279,22 +279,32 @@ var launchdConfig = `<?xml version='1.0' encoding='UTF-8'?>
 <key>Label</key><string>{{html .Name}}</string>
 <key>ProgramArguments</key>
 <array>
-        <string>{{html .Path}}</string>
-{{range .Config.Arguments}}
-        <string>{{html .}}</string>
-{{end}}
-</array>
-{{if .UserName}}<key>UserName</key><string>{{html .UserName}}</string>{{end}}
-{{if .ChRoot}}<key>RootDirectory</key><string>{{html .ChRoot}}</string>{{end}}
-{{if .WorkingDirectory}}<key>WorkingDirectory</key><string>{{html .WorkingDirectory}}</string>{{end}}
-{{if .Environment}}<key>EnvironmentVariables</key>
+	<string>{{html .Path}}</string>
+{{range .Config.Arguments}}	<string>{{html .}}</string>
+{{end}}</array>
+{{if .UserName}}<key>UserName</key><string>{{html .UserName}}</string>
+{{end}}{{if .ChRoot}}<key>RootDirectory</key><string>{{html .ChRoot}}</string>
```

Comment:
Why is then end moved in this way?

Reply:
The string formatting isn't that nice looking if you don't include the newline in the if statement (there's a ton of random blank space in the plist file).